### DISCUSSION TITLE
Interpretación de condiciones individuales Recall/F1score

### DISCUSSION THREAD

**Discussion by PabloMoyaGalvez** [dlopezg/mvpalab#12] [General]
Hola David, voy a tomar el atrevimiento de consultarte sobre la interpretación de unos datos, ya que tengo dudas sobre como interpretar individualmente la generalización temporal en el caso del Recall, entiendo que el recall es principalmente la sensibilidad de la prueba. Pero al ver la generalización temporal en los resultados de las distintas condiciones me cuesta interpretarlos ya que no sé  de que forma se relacionan las condiciones al observarlas de forma independiente en la generalización temporal. Por ejemplo adjunto una imagen de estímulos frecuentes(condición a) y oddball(condición b) al ver el recall  es muy sensible en la condición frecuente a diferencia del oddball el cual se presenta principalmente en la diagonal, pero presenta significancia inversa en la generalización temporal, ¿es correcta esta aproximación?. ¿Es posible afirmar que el estimulo frecuente (condición a) describe una fuerte sensibilidad para esta condición en el grupo 2 y que la condición b es más sensible en un periodo más definido en el tiempo en el grupo 2?

Me puedes ayudar a como puedo interpretar de forma independiente estas condiciones. Adjunto imagen, y de antemano muchas gracias, y espero no incomodar.

Saludos Atte
Pablo

![Captura de pantalla 2023-04-21 003318](https://user-images.githubusercontent.com/130928368/233542422-fe3566ba-d9f2-4230-8e12-e0d539c0349a.png)

**Comment by dlopezg** [comment]
Buenos días Pablo,

Los resultados de generalización temporal debes interpretarla usando la media para cada grupo o alguna otra métrica que no segregue por condiciones (como la ACC o el AUC). 

Las gráficas individuales para cada condición te pueden dar una idea de cómo los clasificadores están funcionando para cada condición, pero no para interpretar tus resultados a nivel de grupo. Me explico. Cuando obtenemos un resultado de clasificación de 0,5 quiere decir que solo la mitad de nuestros ensayos se están clasificando correctamente, pero eso puede ser resultado de varias situaciones. Por ejemplo, que todos los ensayos de la condición A se estén clasificando de forma correcta y que todos los ensayos de la condición b se estén clasificando de forma incorrecta. Esto, de media, si tienes el mismo número de ensayos por condición te daría un resultado de 0,5. Otra posible opción sería que la mitad de tus ensayos, tanto de la condición A como de la condición B, se estén clasificando de forma incorrecta. Ambas situaciones hacen que el clasificador no generalice en el tiempo, pero la situación que genera esta falta de generalización es distinta.

Para saber como interpretar los resultados de generalización te recomiendo que te leas este artículo:

https://www.sciencedirect.com/science/article/pii/S1364661314000199

Como consejo también te diría que al representar las gráficas para los dos grupos ajustes la escala de color a los mismos valores para ambas, de esta forma podrás ver mejor en qué grupo la generalización temporal es mas fuerte. 

Espero haberte ayudado, 

Un saludo,
David
